great-idea/services.html

```diff
@@ -16,92 +16,124 @@
 
 <body>
 
-
-Services
-Product
-Vision
-Features
-About
-Contact
-
-<img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-
-
+    <div class="navigation">
+      <nav> 
+          <a href="services.html">Services</a>
+          <a>Product</a>
+          <a>Vision</a>
+          <a>Features</a>
+          <a>About</a>
+          <a>Contact</a>
+          <div> 
+            <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
```

Comment:
Please, 'Wrap the great idea logo in an anchor tag and point the href to index.html'